.github/workflows/scripts/build_pr.sh

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env bash
+
+export MY_ORG=$(echo "${GITHUB_REPOSITORY}" | awk -F / '{print $1}')
```

Comment:
See https://github.com/kbase/narrative/pull/2202 for the PR where this script first appeared; I've cherry-picked the commit over and updated the rest of the files to use it.